terraform/templates/ec2_user_data.sh

```diff
@@ -44,6 +45,7 @@ ${host_log_path} {
 	copytruncate
 }
 EOF
+sudo mv /etc/cron.daily/logrotate /etc/cron.hourly/
```

Comment:
I didn't see where this is defined.

Reply:
@sausagee I think this is the logrotate default, it's not defined anywhere.